README.md

```diff
@@ -15,32 +16,19 @@ current directory. This may be overriden using the `-f` option.
 
 Example: `function-runner -f '../my-function-name.wasm' '../my-input.json'`
 
-## Commands (optional)
-
-* `cargo install --path .` : Build and install the `function-runner` command.
-* `function-runner` : Execute Functions.
-
-## How to use this repo
+## Development
 
 Building requires a rust toolchain of at least `1.56.0` (older may work). `cargo install --path .` will build
 and add the `function-runner` command to your path.
 
-### Usage
+### Commands
 
-```
-$ function-runner --help
-function-runner 3.0.0
-Simple function runner which takes JSON as a convenience
+- `cargo install --path .` : Build and install the `function-runner` command.
+- `function-runner` : Execute functions.
 
-USAGE:
-    function-runner [OPTIONS] <INPUT>
+## Usage
 
-ARGS:
-    <INPUT>    Path to json file containing function input
+If you wish to use `function-runner` without compiling it, the [Releases](https://github.com/Shopify/function-runner/releases) page
```

Comment:
I think this might be better if it's above the `## Development` section and under a dedicated installation section so it's more immediately obvious. Ideally someone just wanting to use the tool shouldn't have to read the whole README.